changelogs/fragments/filetree_create_api_plugin.yml

```
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - Set the actual plugin to connect to the gateway API
```

Can we be more descriptive as well where the change is?
```suggestion
  - Set the lookup plugin in filetree_create to use the ansible.platform.gateway_api plugin to capture the AAP version
```